lib/nerve/reporter/zookeeper.rb

```diff
@@ -124,6 +125,21 @@ def host_list_to_cluster(list)
       last_non_number ? first_token[0..last_non_number] : first_host
     end
 
+    def get_key(service)
+      if service.has_key?('use_path_encoding') && service['use_path_encoding'] == true
+        obj = {
+          'host' => service['host'],
+          'port' => service['port']
+        }
+        if service.has_key?('labels')
+          obj['labels'] = service['labels']
+        end
+        '/base64_' + Base64.urlsafe_encode64(JSON(obj)) + '_'
```

Comment:
Why skip encoding `name` field? `host` is `IP` or `CNAME`, they are not debuggable friendly